tests/test_account.py

```diff
@@ -5,11 +5,21 @@
 
 import bs4
 import pytest
-from action.models import Action
-from django.core.files.uploadedfile import SimpleUploadedFile
 from django.urls import reverse
 from userauth.models import CustomUser
-from userauth.util import get_system_user, slug_to_user, user_to_slug
+
+
+@pytest.fixture(scope="session")
+def celery_worker_parameters():
+    """Disable the ping check during testing
+
+    Otherwise get this error:
+        "celery.exceptions.NotRegistered: 'celery.ping'"
+
+    Issue https://github.com/celery/celery/issues/4851
+    This "fix" https://github.com/celery/celery/issues/3642#issuecomment-457773294
+    """
+    return {"perform_ping_check": False}
```

Comment:
I'm not sure why I was suddenly getting this area, so this is just a "make it go away" kind of fix, without really understand how celery works during tests....

Reply:
Hmm interesting, maybe when testing, celery tries to ping the redis broker to verify existence but fails because pytest doesn't start celery too. I think the fix isn't too bad, we probably don't test celery functionality.